packages/zone/src/make-once.js

```diff
@@ -0,0 +1,74 @@
+// @ts-check
+const { Fail } = assert;
+
+/**
+ * @param {string} baseLabel
+ * @param {import('.').Stores} stores
+ * @param {MapStore<string, any>} [backingStore]
+ */
+export const makeOnceKit = (baseLabel, stores, backingStore) => {
```

Comment:
```suggestion
export const makeOnceKit = (baseLabel, stores, backingStore = undefined) => {
```
Do you not like this better? Am I fighting a losing battle on this one?

Reply:
Oh!  I thought that was conflicting with the JSDoc types, but I was mistaken.  Yes, I like it better.